wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/devfile/validator/DevfileSchemaValidatorTest.java

```diff
@@ -85,6 +86,24 @@ public void shouldThrowExceptionOnValidationOfNonValidDevfile(
     fail("DevfileFormatException expected to be thrown but is was not");
   }
 
+  @Test
+  public void shouldThrowExceptionWhenDevfileHasUnsupportedApiVersion() throws Exception {
+    try {
+      String devfile =
+          "---\n" + "apiVersion: 111.111\n" + "metadata:\n" + "  name: test-invalid-apiversion\n";
+      schemaValidator.validateYaml(devfile);
+    } catch (DevfileFormatException e) {
+      assertEquals(
+          e.getMessage(),
+          "Version '111.111' of the devfile is not supported. "
+              + "Supported versions are '"
```

Comment:
`Constants.SUPPORTED_VERSIONS` is a list. I would personally not wrap it in single-quotes here.